src/Query/index.ts

```diff
@@ -256,7 +257,7 @@ export class Query {
     return agg;
   }
 
-  getAggregationQuery(searchProperty: string[] | null, filters: Filter<Document>[], page?: number): any[] {
+  getAggregationQuery(searchProperty: string[] | null, filters: Filter<Document>[], page?: number): Filter<Document>[] {
```

Comment:
same as noted below, `collection.aggregate` expects a Document[] type to be passed, as opposed to Filter<Document> 

https://github.com/mongodb/node-mongodb-native/blob/main/src/collection.ts#L936

Reply:
to be clear, a Mdb.Document 😅

Reply:
haha was just about to ask. seems like all in this file the `Filter` type is being cast to a `SearchIndex/types.ts/Document` type, should this all be changed to `mongodb.Document`?